CHANGELOG.md

```diff
@@ -7,9 +7,21 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [0.0.4] - 2020-01-23
+### Added
 
-## [0.0.3] - 2020-01-17
+- `LocaleSwitcher` dependency to handle locale query strings
+- Internationalization
+- Max length, and required validations
+- Added mock for `AddressContext`
+- Added JSON in new format for `ARG`, `BRA` and `USA`, and their TS interface definitions
+
+### Changed
+
+- `AddressForm` now contains an instance of `PlaceDetails`, and an edit button
+- `PlaceDetails` and `AddressForm` now consume `address` information directly from `AddressContext` orchestrator
+- Simplified `PlaceDetails` tests
+
+## [0.0.4] - 2020-01-17
```

Comment:
did you accidentally change this date? the diff above says the `v0.0.4` was released on `2020-01-23` instead of `2020-01-17`, which is the date `v0.0.3` was released

Reply:
It was on purpose, that `v0.0.4` previously released was actually empty.